code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -28,33 +28,33 @@
 /obj/structure/closet/crate/can_close()
 	return 1
 
-/obj/structure/closet/crate/open()
-	if(src.opened)
-		return 0
-	if(!src.can_open())
-		return 0
+/obj/structure/closet/crate/open(mob/user)
+	if(opened)
+		return
+	if(!can_open())
+		return
 
 	if(rigged && locate(/obj/item/radio/electropack) in src)
-		if(isliving(usr))
-			var/mob/living/L = usr
-			if(L.electrocute_act(17, src))
+		if(iscarbon(user))

```

Comment:
With the way dynamic typecasting works in BYOND, this won't change anything at all.

Reply:
Oh okay, serves me right for not checking an issue actually exists before "fixing" it 😄